docs/source/conf.py

```diff
@@ -73,6 +73,10 @@
 # source_suffix = ['.rst', '.md']
 source_suffix = ".rst"
 
+linkcheck_ignore = [
+    r"https://github\.com/.+?#.+",
```

Comment:
Why ignore github.com links?

Reply:
They are generated by frontend rendering, so they can't be found by the link checker.